diff --git a/node_modules/react-number-format/dist/react-number-format.cjs.js b/node_modules/react-number-format/dist/react-number-format.cjs.js
index 754e19b..0110657 100644
--- a/node_modules/react-number-format/dist/react-number-format.cjs.js
+++ b/node_modules/react-number-format/dist/react-number-format.cjs.js
@@ -237,7 +237,7 @@ function setCaretPosition(el, caretPos) {
     // ^ this is used to not only get 'focus', but
     // to make sure we don't have it everything -selected-
     // (it causes an issue in chrome, and having it doesn't hurt any other browser)
-    if (el !== null) {
+    if (el !== null && typeof el.value !== 'undefined' && typeof el.focus === 'function') {
         /* @ts-ignore */
         if (el.createTextRange) {
             /* @ts-ignore */
@@ -248,12 +248,12 @@ function setCaretPosition(el, caretPos) {
         }
         // (el.selectionStart === 0 added for Firefox bug)
         if (el.selectionStart || el.selectionStart === 0) {
-            el.focus();
+            el.focus?.();
             el.setSelectionRange(caretPos, caretPos);
             return true;
         }
         // fail city, fortunately this never happens (as far as I've tested) :)
-        el.focus();
+        el.focus?.();
         return false;
     }
 }
diff --git a/node_modules/react-number-format/dist/react-number-format.es.js b/node_modules/react-number-format/dist/react-number-format.es.js
index 02e19e2..bf2700e 100644
--- a/node_modules/react-number-format/dist/react-number-format.es.js
+++ b/node_modules/react-number-format/dist/react-number-format.es.js
@@ -241,12 +241,12 @@ function setCaretPosition(el, caretPos) {
         }
         // (el.selectionStart === 0 added for Firefox bug)
         if (el.selectionStart || el.selectionStart === 0) {
-            el.focus();
+            el.focus?.();
             el.setSelectionRange(caretPos, caretPos);
             return true;
         }
         // fail city, fortunately this never happens (as far as I've tested) :)
-        el.focus();
+        el.focus?.();
         return false;
     }
 }
diff --git a/node_modules/react-number-format/dist/react-number-format.js b/node_modules/react-number-format/dist/react-number-format.js
index 26f7c36..35e2605 100644
--- a/node_modules/react-number-format/dist/react-number-format.js
+++ b/node_modules/react-number-format/dist/react-number-format.js
@@ -247,12 +247,12 @@
             }
             // (el.selectionStart === 0 added for Firefox bug)
             if (el.selectionStart || el.selectionStart === 0) {
-                el.focus();
+                el.focus?.();
                 el.setSelectionRange(caretPos, caretPos);
                 return true;
             }
             // fail city, fortunately this never happens (as far as I've tested) :)
-            el.focus();
+            el.focus?.();
             return false;
         }
     }
